clients/include/testing_gemm_strided_batched.hpp

```diff
@@ -44,8 +44,22 @@ hipblasStatus_t testing_gemm_strided_batched(const Arguments& argus)
 
     int A_size, B_size, C_size, A_row, A_col, B_row, B_col;
     int bsa, bsb, bsc; // batch size A, B, C
-    T   alpha = argus.alpha;
-    T   beta  = argus.beta;
+
+    float alpha_float = argus.alpha;
+    float beta_float  = argus.beta;
+
+    T alpha, beta;
+
+    if(is_same<T, hipblasHalf>::value)
+    {
+        alpha = float_to_half(alpha_float);
+        beta  = float_to_half(beta_float);
+    }
+    else
+    {
+        alpha = static_cast<T>(alpha_float);
+        beta  = static_cast<T>(beta_float);
+    }
```

Comment:
Not for this release but expect this should move to use helpers the same as rocblas.  If you can get_alpha<T> and get_beta<T> you can handle this with specializations along with NaN support.